articles/azure/azs-how-create-S2S-VPN.md

```diff
@@ -0,0 +1,222 @@
+
+### Intended audience
+
+To complete the steps in this guide, you must have appropriate access to a subscription in the UKCloud Azure Stack portal.
+
+## Create a virtual network
+
+First, a virtual network will need to be created. This virtual network will be able to send and receive traffic through the virtual network gateway. A virtual network can not be associated with more than one gateway.
+
+1. Log in to the UKCloud Azure Stack portal.
+
+   For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md)
+
+2. In the favourites panel, select **Create a resource**.
+
+   ![Create a resource option in favourites panel](images/azsp_newmenu.png)
+
+3. In the **New** blade, select **Networking**.
+
+   ![Networking option in New blade](images/azs-browser-new-networking.png)
+
+4. In the **Featured** section, select **Virtual network**.
+
+   ![Create new networking resource](images/azs-browser-networking-create.png)
+
+5. In the **Create virtual network** blade, enter the following information:
+
+   **Name** - The name of the virtual network.
+
+   **Address Space** - The virtual network's address range in CIDR notation.
+
+   **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   **Location** - This will be "frn00006", which is the location of the Azure Stack.
+
+   **Subnet Name** - The name of the first subnet within the virtual network.
+
+   **Address Range** - The subnet's address range in CIDR notation (e.g. 192.168.1.0). It must be contained by the address space of the virtual network. The address range of a subnet which is in use can't be edited.
+
+   ![Create new virtual network](images/azs-browser-create-virtual-network.png)
+
+6. Click **Create**.
+
+7. You can monitor the progress of your virtual network's deployment by clicking the **Notifications** icon.
+
+   ![!Notification showing VM deployment in progress](images/azsp_createvm_progress.png)
+
+8. Click the deployment notification for the deployment to view details of the deployment of the virtual network.
+
+9. When the deployment is finished, the notification wil change to **Deployment succeeded**.
+
+10. After your virtual network has deployed, you can view it by clicking **All services** in the favourites panel, then selecting **Virtual networks** under the networking section.
+
+11. Select your virtual network from the list.
+
+12. In the blade for your virtual network, you can view and monitor the virtual network, change its settings and perform diagnostics and troubleshooting.
+
+## Create the gateway subnet
+
+To associate a virtual network with a gateway, it must first contain a valid gateway subnet. The following steps outline how to create a gateway subnet.
+
+1. In the portal, navigate to the virtual network which was created previously.

```

Comment:
In the portal, navigate to the virtual network that you created in the previous section.

Reply:
Fixed